.github/workflows/github-actions.yml

```
@@ -50,16 +50,16 @@ jobs:
           push: true
           tags: ${{ secrets.DOCKER_REPO }}:latest
 
-      - name: Deploy to Server
-        uses: appleboy/ssh-action@master
+      - name: AWS credentials
+        uses: aws-actions/configure-aws-credentials@v2
```

actionlint (1.7.7)

the runner of "aws-actions/configure-aws-credentials@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)